twilio/task_router/workflow_config.py

```diff
@@ -21,6 +21,9 @@ def to_json(self):
 
     @staticmethod
     def json2obj(data):
+        # replace instances of "filter_friendly_name" with "friendly_name"
+        data = data.replace("filter_friendly_name", "friendly_name")
```

Comment:
this is very dangerous. If the user has filter_friendly_name in the expression or some other value, this will break their software.